_config.yml

```diff
@@ -40,7 +40,7 @@ google_analytics:
 
 # Your website URL (e.g. http://barryclark.github.io or http://www.barryclark.co)
 # Used for Sitemap.xml and your RSS feed
-url: https://bhsdevilbotz.com/
+url: https://devilbotz.org/
```

Comment:
Think I'll need to leave this PR open until the redirect is complete. It also involves renaming the Github web page folder, which I'm not sure will be seamless. I'm away the rest of this week (Oct 11-Oct 16).

Reply:
I think it's possible to switch this without messing with the domain forwarding.  I currently have devilbotz.org redirecting to bhsdevilbotz.com, also using github pages.  It should be simple to switch them around.  Maybe it'll "just work"

Reply:
@L5eoneill, success! I was able to change the this website to use devilbotz.org instead, and now have bhsdevilbotz.org doing an HTTP redirect to devilbotz.org.  No need to change anything with the domain registrar.  See this repo, which now handles bhsdevilbotz.com and implements the redirect: https://github.com/DevilBotz2876/bhsdevilbotz.com

Reply:
BTW, the following URL forwarding is broken:
- [ ] bhsdevilbotz.org
- [ ] team2876.com

Do we care/need these anyway?